packages/editor/src/managers/clipboard/paste-manager.ts

```diff
@@ -197,8 +197,13 @@ export class PasteManager {
         selectedBlock?.text?.insertList(insertTexts, endIndex);
         selectedBlock &&
           this._addBlocks(blocks[0].children, selectedBlock, 0, addBlockIds);
-
-        parent && this._addBlocks(blocks.slice(1), parent, index, addBlockIds);
+        if (blocks[0].flavour === 'affine:divider') {
```

Comment:
Maybe you need to add some comments to explain why the divider is special

Reply:
> And please stick to the `matchFlavour` pattern.

But the block here is an OpenBlockInfo[], not a model